tests/BaseAppTest.php

```diff
@@ -59,6 +68,21 @@ public function testBoot(): void
         $this->app->boot();
     }
 
+    public function testBootCalledTwice(): void
```

Comment:
Why do we need to test this case?
What can happen here?

Reply:
we test the early return statement here:

https://github.com/starweb/starlit-app/blob/780f28b8faa843a0446718f05b45fd6c8124ce67/src/BaseApp.php#L136-L138